server/src/api/label.ts

```diff
@@ -20,8 +21,8 @@ router.get('/', authenticateToken, async (req: AuthorizedRequest, res) => {
 router.post('/', authenticateToken, async (req: AuthorizedRequest, res) => {
   const { userIdx } = req.user;
   const { title, color, unit } = req.body;
-  await executeSql('insert into label (title, color, unit, user_idx) values (?, ?, ?, ?)', [title, color, unit, userIdx]);
-  res.sendStatus(200);
+  const result = (await executeSql('insert into label (title, color, unit, user_idx) values (?, ?, ?, ?)', [title, color, unit, userIdx])) as OkPacket;
+  res.status(200).send({ idx: result.insertId });
```

Comment:
```suggestion
  res.status(201).send({ idx: result.insertId });
```
<br>

[201번 상태 코드](https://developer.mozilla.org/ko/docs/Web/HTTP/Status/201)